### ISSUE TITLE
Compatibility update for Modo 11 to 12

### ISSUE THREAD

**Issue by smoluck** [tcrowson/Modo_Popups#1]
Hi Tim,

i'm q fan of your kit, qnd qs i wanted to get popups loaded also under modo11 to 12, i've created a modified one, here.

https://gum.co/qhYiU

Feel free to add it to your GitHub. It fix the compatibility with Modo11 to 12.

hope you are aware that i'm not reselling it by any way. just wanted to make it compatible.

Best regards. Franck

**Comment by tcrowson** [comment]
Thanks so much Franck! I don’t use modo anymore and don’t even have access
to modo 11/12 to test, so I appreciate this very much.

**Comment by tcrowson** [comment]
Franck, it seems like the changes are mainly to the config, to support later versions beyond 901. Correct? I have posted the changes, but I am unable to test the result. Can you confirm if it all works as intended now?

**Comment by smoluck** [comment]
yes is was only change to CFG and index. I will take a look to test it. glad to help you.

can i connect to you on LinkedIn, Twitter or Facebook ?

**Comment by smoluck** [comment]
Uppdate confirmed.

it's working with your update too.
